### PR TITLE
Add PHPCS to core integration tests 

### PR DESCRIPTION
**Pull request by laemtl**
Replaces #6149

### CODE REVIEW

File: test/run-php-linter.sh [hunk 1]
```diff
@@ -52,8 +52,8 @@ vendor/bin/phpcs --standard=test/LorisCS.xml --extensions=php,inc \
     php/ \
     htdocs/ \
     modules/ \
+    "test/integrationtests/" \
```
Comment (driusan) [inline test/run-php-linter.sh:55]: The variable "test_list" above doesn't appear to be used anymore and should be deleted.

Comment (laemtl) [reply test/run-php-linter.sh:55]: @johnsaigle Any reason why PHPCS is not run on the test_list anymore?

Comment (driusan) [reply test/run-php-linter.sh:55]: You changed it to run on the entire directory that had the files in the list.

Comment (laemtl) [reply test/run-php-linter.sh:55]: Oh, missed that. You're right then.

**Pull request merged**